pkg/pillar/types/zedroutertypes.go

```diff
@@ -2875,3 +2909,28 @@ func (status OnboardingStatus) LogDelete(logBase *base.LogObject) {
 func (status OnboardingStatus) LogKey() string {
 	return string(base.OnboardingStatusLogType) + "-" + status.Key()
 }
+
+// AppNumber :
+// PS. Any change to AppNumMax, must be
+// reflected in the BitMap Size(8 bytes)
+const (
+	AppNumStart   = 0   // DhcpRange start
+	AppNumEnd     = 253 // DhcpRange end
+	AppNumMax     = 254 // Max Count
+	AppNumBridge  = -2  // For Bridge
+	AppNumInvalid = -1
+)
+
+// NetworkInstanceAppAddressMap :
+type NetworkInstanceAppAddressMap struct {
+	NetworkID string
+	AppID     string
+	AppNum    int
```

Comment:
Presumably this should be called AppNumOnNet to avoid confusing it with the AppNum used elsewhere.